functions.yml

```diff
@@ -387,11 +406,13 @@ gwas_functions:
         - name: genotypes
           doc: An array of genotypes
```

Comment:
A numeric array of genotypes (to distinguish for the user from genotype structs)

Reply:
Fixed